applications/Preprocessor/lib/mesh/MeshImpl/Mesh.h

```diff
@@ -184,7 +185,7 @@ class Mesh {
         dimension>>&
         getEntities() const;
     template <int entityDimension>
-    MeshEntity<entityDimension, dimension> getEntity(EntityGId i) const {
+    MeshEntity<entityDimension, dimension>& getEntity(EntityGId i) {
```

Comment:
maybe also want a `const`version

Reply:
Most of this I treat as a problem when I need it, as I'm not sure about the pass by value/reference of most of these functions.